content/en/docs/setup/install/multicluster/shared-vpn/index.md

```diff
@@ -79,6 +79,7 @@ cluster. You can install the component in one of two ways:
     $ istioctl manifest apply \
     --set profile=remote \
     --set values.global.controlPlaneSecurityEnabled=false \
+    --set values.global.createRemoteSvcEndpoints=true \
```

Comment:
Should this value also be added to the `Installation configuration parameters` table?

Reply:
oh yes, I just updated it, also added to some other places